opennmt/utils/scorers.py

```diff
@@ -95,6 +170,18 @@ def make_scorers(names):
       scorer = BLEUScorer()
     elif name == "rouge":
       scorer = ROUGEScorer()
+    elif name == "wer":
+      scorer = WERScorer()
+    elif name == "ter":
+      scorer = TERScorer()
+    elif name == "precision":
+      scorer = PRECISIONScorer()
+    elif name == "recall":
+      scorer = RECALLScorer()
+    elif name == "fmeasure":
+      scorer = FMEASUREScorer()
```

Comment:
I'm wondering if we need these scores as `PRFScorer` computes all values in one shot. What do you think?

Reply:
Actually, it was to have separate scores only, but yes, the last scorer could be used.